.github/workflows/coverage.yml

```diff
@@ -31,6 +31,12 @@ jobs:
           arch: x86_64
           script: bash contrib/instrumentation.sh
 
+      - name: Upload screenshot result
+        uses: actions/upload-artifact@v1
```

Comment:
v2 of this was released and it sounds like it sucks waaaaaaaay less

Reply:
I'll update it.